js/renovation/widget.tsx

```diff
@@ -254,6 +257,13 @@ export default class Widget extends JSXComponent<WidgetInput> {
         return () => dxClick.off(this.widgetRef, { namespace });
     }
 
+    @Effect()
+    contentReadyEffect() {
+        const { onContentReady } = this.props;
+
+        onContentReady?.({});
```

Comment:
Should we pass here the following [object](https://js.devexpress.com/Documentation/20_1/ApiReference/UI_Widgets/dxButton/Configuration/#onContentReady)?

`component, element, model`?

Reply:
No, ContentReady is an action that passes these parameters

Reply:
https://github.com/DevExpress/DevExtreme/pull/12174/files#diff-03a401c56907a4c058f12b32b61523fdR56